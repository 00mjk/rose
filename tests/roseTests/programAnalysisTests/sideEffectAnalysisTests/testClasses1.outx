Call side effects on project: 
Function  Side Effects ::foo::ptrMod: x 
Statement Side Effects testClasses1.C-7-28-261: x 
Function  Side Effects ::foo::refMod: x 
Statement Side Effects testClasses1.C-8-27-261: x 
Function  Side Effects ::foo::globalMod: local ::global 
Statement Side Effects testClasses1.C-9-39-261: local 
Statement Side Effects testClasses1.C-9-51-261: ::global 
Function  Side Effects main: x y ::global 
Statement Side Effects testClasses1.C-18-5-374: x 
Statement Side Effects testClasses1.C-19-5-374: y 
Statement Side Effects testClasses1.C-20-5-374: ::global 
Call side effects on file: 
Function  Side Effects ::foo::ptrMod: x 
Statement Side Effects testClasses1.C-7-28-261: x 
Function  Side Effects ::foo::refMod: x 
Statement Side Effects testClasses1.C-8-27-261: x 
Function  Side Effects ::foo::globalMod: local ::global 
Statement Side Effects testClasses1.C-9-39-261: local 
Statement Side Effects testClasses1.C-9-51-261: ::global 
Function  Side Effects main: x y ::global 
Statement Side Effects testClasses1.C-18-5-374: x 
Statement Side Effects testClasses1.C-19-5-374: y 
Statement Side Effects testClasses1.C-20-5-374: ::global 
Call side effects within for loop: 
