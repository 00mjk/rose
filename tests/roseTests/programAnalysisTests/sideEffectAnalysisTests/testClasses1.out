Call side effects on project: 
Function  Side Effects foo::ptrMod: x 
Statement Side Effects testClasses1.C-134839016: x 
Function  Side Effects foo::refMod: x 
Statement Side Effects testClasses1.C-134846616: x 
Function  Side Effects foo::globalMod: local ::global 
Statement Side Effects testClasses1.C-134858808: local 
Statement Side Effects testClasses1.C-134859736: ::global 
Function  Side Effects main: x ::global y 
Statement Side Effects testClasses1.C-134876248: x 
Statement Side Effects testClasses1.C-134877608: y 
Statement Side Effects testClasses1.C-134878856: ::global 
Call side effects on file: 
Function  Side Effects foo::ptrMod: x 
Statement Side Effects testClasses1.C-134839016: x 
Function  Side Effects foo::refMod: x 
Statement Side Effects testClasses1.C-134846616: x 
Function  Side Effects foo::globalMod: local ::global 
Statement Side Effects testClasses1.C-134858808: local 
Statement Side Effects testClasses1.C-134859736: ::global 
Function  Side Effects main: x ::global y 
Statement Side Effects testClasses1.C-134876248: x 
Statement Side Effects testClasses1.C-134877608: y 
Statement Side Effects testClasses1.C-134878856: ::global 
Call side effects within for loop: 
